### PR TITLE
2022-06-08 Ask the Hybrid Brains

### PR DESCRIPTION
**Pull request by JakeSidSmith**
Waiting on meetup link before merging.

Screen Shot 2022-06-01 at 16 34 04

### CODE REVIEW

File: _posts/2022-06-08-ask-the-hybrid-brains.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+online: true
+venue:
+  name: PLATF9RM
+  link: https://www.platf9rm.com/
+  location: https://goo.gl/maps/pkoNTTRA27bh5GKD8
+  address: Floor 5, Tower Point, 44 North Road, Brighton, BN1 1YR
+  latlong: 50.825944,-0.1437302
+# draft: true # Delete this line to publish
+---
+We're back! You will be happy to know that you will finally have the opportunity to come face-to-face with like-minded developers and enthusiasts (at Async at least) for the first time in over 2 years! Can't make it in-person? Don't worry, you'll also be able to join us online.
```
Comment (orangespaceman) [inline _posts/2022-06-08-ask-the-hybrid-brains.md:27]: "You will be happy" - sounds forceful! 

I would consider rephrasing it along the lines of "we are happy to announce" although that's also how the next paragraph starts, so...

File: _posts/2022-06-08-ask-the-hybrid-brains.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+I am very happy to announce that on the evening of Wednesday 8th June, we will be hosting an in-person hybrid version of our popular __"Ask the Brains"__ event for the first time. This means that you will have the ability to join us in-person at [PLATF9RM](https://goo.gl/maps/pkoNTTRA27bh5GKD8) in Brighton, or to join us online!
```
Comment (orangespaceman) [inline _posts/2022-06-08-ask-the-hybrid-brains.md:29]: "We" rather than "I"

File: _posts/2022-06-08-ask-the-hybrid-brains.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+Inspired by the monthly Brighton [ALT.NET meetup](http://brightonalt.net/), we will be asking/talking/ranting about everything web-related - 5 - 10 minutes per topic.
```
Comment (orangespaceman) [inline _posts/2022-06-08-ask-the-hybrid-brains.md:31]: I like "ranting" but it's maybe not so welcoming! Especially as we say it's trying to be a friendly environment in the next para

**Review by orangespaceman — APPROVED** [review]
Looks great! Feel free to ignore these comments...

File: _posts/2022-06-08-ask-the-hybrid-brains.md (hunk 2 above)
Comment (JakeSidSmith) [reply _posts/2022-06-08-ask-the-hybrid-brains.md:29]:
```suggestion
We are very happy to announce that on the evening of Wednesday 8th June, we will be hosting an in-person hybrid version of our popular __"Ask the Brains"__ event for the first time. This means that you will have the ability to join us in-person at [PLATF9RM](https://goo.gl/maps/pkoNTTRA27bh5GKD8) in Brighton, or to join us online!
```

**Pull request merged**